ci/cspell.json

```diff
@@ -5,6 +5,7 @@
   "language": "en",
   // words - list of words to be always considered correct
   "words": [
+    "absl",	  
```

Comment:
I think we expected this to be spelled in backticks for comments, .i.e. 

```
// `absl::Foo`
```

Reply:
Thanks, good to know.